### DISCUSSION TITLE
What is your system?

### DISCUSSION THREAD

**Discussion by gbakeman** [nutdotnet/Coco.Nut#5] [Polls]
For those coming from WinNUT, what operating syatem are you using? What operating syatems or devices would you like to see your UPS informatiom on?

**Comment by rederikus** [comment]
I would hve liked to have voted but I have Windows 11, 10 and Linux.  I also use an Android phone.  However the poll onlly allows for a single choice.

**Reply by gbakeman** [reply]
Good point - I wish I could have made the poll multiple choice, although I don't see an option for that. Thank you for your feedback.
